test/parallel/test-http-upgrade-server.js

```diff
@@ -100,7 +100,7 @@ function test_upgrade_with_listener() {
 
     if (state === 1) {
       assert.strictEqual(data.substr(0, 12), 'HTTP/1.1 101');
-      assert.strictEqual(request_upgradeHead.toString('utf8'), 'WjN}|M(6');
+      assert.strictEqual(request_upgradeHead.toString('utf8'), '');
```

Comment:
I think this assertion can be removed completely. It is no longer relevant.
I hope other tests ensure that upgrade request body is not lost.